### PR TITLE
Ability to extract Operations, Deposits and Withdraws

### PR DESCRIPTION
**Pull request by dnfd**

### CODE REVIEW

File: app/api/v2/admin/deposits.rb [hunk 1]
```diff
@@ -7,6 +7,8 @@ module Admin
       class Deposits < Grape::API
         helpers ::API::V2::Admin::Helpers
 
+        content_type :csv, 'text/csv'
```
Comment (ysv) [inline app/api/v2/admin/deposits.rb:10]: does it add content_type format to all /deposits endpoints ? 
If yes then I disagree that we need it on post '/deposits/new' & post '/deposits/actions'

Add specs please

Comment (dnfd) [reply app/api/v2/admin/deposits.rb:10]: > does it add content_type format to all /deposits endpoints ?

Yes, it applies to all endpoints, did not find approach better then separating 'csv' endpoint in another controllers

File: app/models/operation.rb [hunk 2]
```diff
@@ -87,5 +87,20 @@ def balance(currency: nil, created_at_from: nil, created_at_to: nil)
         where(currency: currency).sum('credit - debit')
       end
     end
+
+    def to_csv
```
Comment (mod) [inline app/models/operation.rb:91]: shouldn't we use a serializer and a convention like to_json / as_json
I think as_csv should return an array, and then serializer, I'm sure there is better ways.
render as csv is part of controller, not a model feature. @ysv @dnfd

**Review by mod — APPROVED** [review]

**Review by ysv — COMMENTED** [review]
It would be better to move common CSV helpers into separate module but overall it looks acceptable

**Pull request closed without merging**